ledger/internal/prefetcher/prefetcher_test.go

```diff
@@ -71,9 +76,15 @@ func (l *prefetcherTestLedger) LookupApplication(rnd basics.Round, addr basics.A
 	return ledgercore.AppResource{}, nil
 }
 func (l *prefetcherTestLedger) LookupAsset(rnd basics.Round, addr basics.Address, aidx basics.AssetIndex) (ledgercore.AssetResource, error) {
+	if aidx > 1000000 {
```

Comment:
instead of using a threshold, make a given asset index invalid ( i.e. `const errorTriggeringAssetIndex = 1000000` )